activemq_xml/datadog_checks/activemq_xml/activemq_xml.py

```diff
@@ -61,7 +61,7 @@ def _fetch_data(self, base_url, xml_url, username, password, suppress_errors):
         url = "%s%s" % (base_url, xml_url)
         self.log.debug("ActiveMQ Fetching queue data from: %s" % url)
         try:
-            r = requests.get(url, auth=auth)
+            r = self.http.get(url, auth=auth)
```

Comment:
```suggestion
            r = self.http.get(url)
```

Reply:
Why remove auth ?

Reply:
It's handled by wrapper

https://github.com/DataDog/integrations-core/blob/5db4f29f7fa65cba5dc074b80f669b0c00fdbb8c/activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example#L27-L35
https://github.com/DataDog/integrations-core/blob/5db4f29f7fa65cba5dc074b80f669b0c00fdbb8c/datadog_checks_base/datadog_checks/base/utils/http.py#L107-L110